code/game/machinery/newscaster/newscaster_machine.dm

```diff
@@ -655,7 +655,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/newscaster, 30)
 			existing_authors += GLOB.news_network.redacted_text
 		else
 			existing_authors += iterated_feed_channel.author
-	if((current_user?.account_holder == "Unknown") || (current_user.account_holder in existing_authors) || isnull(current_user?.account_holder))
+	if(!current_user?.account_holder || current_user.account_holder == "Unknown" || (current_user.account_holder in existing_authors))
```

Comment:
this makes AI's completely unable to make news channels.

Reply:
They were already unable to do so

Reply:
In my testing, at least, they couldn't, I'm still not sure how you managed your results.